Assets/Scripts/PackItUp/Controllers/CameraController.cs

```diff
@@ -0,0 +1,46 @@
+using UnityEngine;
+
+namespace PackItUp.Controllers {
+    public class CameraController : MonoBehaviour {
+        [SerializeField] private Transform player1;
```

Comment:
For now I think this works but it might be wise eventually to have you can have this as an array and allow multiple players, maybe we can set up a limit and that's it 👀

Reply:
Same goes for the cameras